Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -432,6 +478,11 @@ namespace Babylon
         return m_frameBufferManager;
     }
 
+    void NativeEngine::Dispose(const Napi::CallbackInfo& info)
+    {
+        m_cancelSource.cancel();
```

Comment:
Why not in the destructor?

Reply:
I just realized we don't even call Dispose in the Playground app. It probably should be in both. Dispose is more immediate.